src/schema/types/inputs/Portfolio.ts

```diff
@@ -22,29 +22,29 @@ export const PortfolioUpdateInput = inputObjectType({
 export const PortfolioSkillInput = inputObjectType({
   name: "PortfolioSkillInput",
   definition(t) {
-    t.nonNull.string("skill"), t.nonNull.int("level");
+    t.nonNull.string("name"), t.nonNull.int("level");
```

Comment:
definition 적는 부분은 다른 코드들은 ;으로 구분되어 있는데 이 파일만 ,로 구분되어 있네요. 다른 부분과 맞추면 더 좋을 것 같아요;D
```suggestion
    t.nonNull.string("name");
    t.nonNull.int("level");
```